pkg/sbproxy/notifications/handlers/broker_handler.go

```diff
@@ -77,11 +77,14 @@ type BrokerResourceNotificationsHandler struct {
 	BrokerClient   platform.BrokerClient
 	CatalogFetcher platform.CatalogFetcher
 
+	SMSettings *sm.Settings
+
```

Comment:
do u need the whole settings?
 maybe better pass platformUsername and platformPassword